framework/core/js/src/forum/components/SecurityPage.tsx

```diff
@@ -17,7 +17,7 @@ import NewAccessTokenModal from './NewAccessTokenModal';
  */
 export default class SecurityPage<CustomAttrs extends IUserPageAttrs = IUserPageAttrs> extends UserPage<CustomAttrs> {
   protected tokens: AccessToken[] | null = null;
-  protected loading: boolean = false;
+  protected loading: null | 'terminate_sessions' | 'global_logout' = null;
```

Comment:
I think we can get away with just using separate loading fields. Terminating other tokens vs global logout shouldn't have race condition implications, and this will give us simpler code